Gruntfile.js

```diff
@@ -33,7 +33,7 @@ module.exports = function (grunt) {
             angular: {
                 src: ['bower_components/angular/angular.min.js',
                       'bower_components/angular-resource/angular-resource.min.js',
-                      'bower_components/angular-route/angular-route.min.js',
+                      'node_modules/angular-ui-router/release/angular-ui-router.min.js',
```

Comment:
As they've mentioned [here](https://ui-router.github.io/ng1/):

> bower releases are considered ‘legacy’

So I decided to use the npm installer.